R/grpslope.R

```diff
@@ -18,9 +18,10 @@
 #
 ###############################################################################
 
-#' @importFrom SLOPE prox_sorted_L1
 #' @importFrom stats lm pchisq qchisq qnorm rnorm sd uniroot coef predict
 #' @importFrom utils head
+#' @importFrom Rcpp sourceCpp
+#' @useDynLib grpSLOPE, .registration = TRUE
```

Comment:
Looks like I should have gotten a Note from `R CMD check` according to <https://stat.ethz.ch/pipermail/r-devel/2017-February/073755.html>, but I'm not getting that note, weird... Will include `.registration = TRUE` in my code anyway just in case (doesn't hurt). Thanks!